README.md

```diff
@@ -68,6 +68,9 @@ To use these scripts, please follow the following steps:
 
 4. **Add ccids**
 
+   - Copy all CCID-GitHubUsername pairs from [GitHub Classroom Roster](https://docs.google.com/spreadsheets/d/1ccueXQgTkS_DWQttuVNnd1wJ1-K7Mp_oFlIvTq5gnrw/edit#gid=0).
+   - Paste them to `student_list.csv`.
+
```

Comment:
This is a public repo and is not specific to this course. I would remove that link from here. It could be in our specific CMPUT 201 admin repo.